Hestia/Hestia/DevicesScreen/UITableViewControllerServerConnect.cs

```diff
@@ -0,0 +1,63 @@
+using Foundation;
+using ObjCRuntime;
+using System;
+using UIKit;
+
+using Hestia.DevicesScreen.resources;
+using Hestia.backend;
+using Hestia.backend.models;
+
+namespace Hestia.DevicesScreen
+{
+
+
+    public partial class UITableViewControllerServerConnect : UITableViewController
+    {
+
+        public UITableViewControllerServerConnect(IntPtr handle) : base(handle)
+        {
+            
+        }
+
+        public override void ViewDidLoad()
+        {
+
+            base.ViewDidLoad();
+            newServerName.Text = "Hestia_Server2018";
+            newIP.Text = "94.212.164.28";
+            newPort.Text = "8000";
+
+        }
+
+		public override bool ShouldPerformSegue(string segueIdentifier, NSObject sender)
+		{
+            if (segueIdentifier == "ServerToDevices")
+            {
+                if (newIP.Text == "94.212.164.28" && newPort.Text == "8000")
```

Comment:
So you only update if the server is the exact same as the old one? 
Why can't we change it, and you might user regex to check the IP input, and port range validation.